data/noiseprofiles.json

```diff
@@ -2647,6 +2647,37 @@
           ]
         },
         {
+          "comment": "dmc-gm5 contributed by axel burri",
```

Comment:
We need to review those PDF graphs generated while creating this profile.
Please upload resulting dt-noiseprofile-XXXXXXXX.tar.gz